mne/io/tests/test_meas_info.py

```diff
@@ -245,6 +245,14 @@ def test_read_write_info(tmpdir):
     m2 = m2.hexdigest()
     assert m1 == m2
 
+    info = read_info(raw_fname)
+    info['meas_date'] = None
+    anonymize_info(info)
```

Comment:
is `info['meas_date'] = None` after this? According to https://mne.tools/dev/generated/mne.io.anonymize_info.html#mne.io.anonymize_info, isn't it going to be reset to Jan 1st 2000, no?

Reply:
It's None, pushed a commit to assert it and updated docstring